### PR TITLE
Patch 9

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Added a new section "Use the new EAC to enable or disable Exchange ActiveSync".

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-exchange-activesync.md [hunk 1]
```diff
@@ -33,9 +33,26 @@ You can use the EAC or Exchange Online PowerShell to enable or disable Microsoft
...
+4. In the **Manage settings for email apps** display pane, do one of the following.
+
+   - To disable Exchange ActiveSync, for the **Mobile (Exchange ActiveSync)** option, when the button is **Enabled**, set to **Disabled**. 
+
+   - To enable Exchange ActiveSync, for the **Mobile (Exchange ActiveSync)** option, when the button is **Disabled**, set to **Enabled**. 
+
+5. Click **Save** to save your change. A message **Email app settings updated successfully** is displayed. Click **Close** to exit.
+
+
+## Use the Classic EAC to enable or disable Exchange ActiveSync
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-exchange-activesync.md:53]:
```suggestion
## Use the classic EAC to enable or disable Exchange ActiveSync
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-exchange-activesync.md [hunk 2]
```diff
@@ -33,9 +33,26 @@ You can use the EAC or Exchange Online PowerShell to enable or disable Microsoft
...
+
+1. In the Classic EAC, navigate to **Recipients** \> **Mailboxes**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-exchange-activesync.md:55]:
```suggestion
1. In the classic EAC, navigate to **Recipients** \> **Mailboxes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**